Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -210,9 +211,10 @@ def self.check_http_content(url, user_agents: [:default])
     "The URL #{url} could use HTTPS rather than HTTP"
   end
 
-  def self.http_content_headers_and_checksum(url, user_agent: :default)
+  def self.http_content_headers_and_checksum(url, hash_needed = false, user_agent: :default)
```

Comment:
@MikeMcQuaid I originally tried that, but it causes `brew audit --online foo` to spit out:
```
Error: wrong number of arguments (2 for 1)
Please report this bug:
  http://docs.brew.sh/Troubleshooting.html
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:214:in `http_content_headers_and_checksum'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:182:in `block in check_http_content'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:181:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:181:in `check_http_content'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1573:in `block in audit_urls'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1565:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1565:in `audit_urls'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1314:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:740:in `block in audit_specs'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:737:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:737:in `audit_specs'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:1259:in `audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:78:in `block in audit'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:74:in `each'
/usr/local/Homebrew/Library/Homebrew/dev-cmd/audit.rb:74:in `audit'
/usr/local/Homebrew/Library/Homebrew/brew.rb:91:in `<main>'
```
Accidentally overlooking the scope creep, using `hash_needed = false` seemed to be the solution. However, I think I am overlooking something and cannot find an actual solution. Any guidance would be most welcome.